lib/elixir_sense/core/erlang_html.ex

```diff
@@ -0,0 +1,192 @@
+defmodule ElixirSense.Core.ErlangHtml do
```

Comment:
I took different approach than ExDoc. In our case translating docs to HTML is not an option. While in some markdown implementations inline HTML is allowed and valid (e.g. GitHub flavoured markdown used by LSP), LSP clients (vide VScode) sanitize it and strip all HTML.